.github/workflows/release.yml

```diff
@@ -15,6 +15,25 @@ jobs:
           eggs link ${{ secrets.CI_NESTLAND_API_KEY }}
           eggs publish
 
-      - name: Send Event to Website to Update Versions
-        run: |
-          curl -XPOST -u "${{ secrets.CI_USER_NAME }}:${{ secrets.CI_USER_PAT }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/drashland/website/dispatches --data '{"event_type": "update_docs", "client_payload": { "module_to_update": "dmm", "version": "${{ github.event.release.tag_name }}" }}'
\ No newline at end of file
+  send-reddit-post:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bluwy/release-for-reddit-action@v1
+        with:
+          username: ${{ secrets.REDDIT_USERNAME }}
+          password: ${{ secrets.REDDIT_PASSWORD }}
+          app-id: ${{ secrets.REDDIT_APP_ID }}
+          app-secret: ${{ secrets.REDDIT_APP_SECRET }}
+          subreddit: deno
+          comment: ${{ github.event.release.body }}
+
+  send-twitter-tweet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ethomson/send-tweet-action@v1
+        with:
+          status: "New Dmm Version Released! https://github.com/drashland/drash/releases/latest"
+          consumer-key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
+          consumer-secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
+          access-token: ${{ secrets.TWITTER_ACCESS_TOKEN }}
+          access-token-secret: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
```

Comment:
I would join everything and call it a *Socials*. Booting a ubuntu VM for each one seems overkill 😃 
```yml
socials:
    runs-on: ubuntu-latest
    steps:
        - uses: bluwy/release-for-reddit-action@v1
          name: Publish to r/deno
          with:
            username: ${{ secrets.REDDIT_USERNAME }}
            password: ${{ secrets.REDDIT_PASSWORD }}
            app-id: ${{ secrets.REDDIT_APP_ID }}
            app-secret: ${{ secrets.REDDIT_APP_SECRET }}
            subreddit: deno
            comment: ${{ github.event.release.body }}
        - uses: ethomson/send-tweet-action@v1
          name: Publish to twitter
          with:
            status: "New Dmm Version Released! https://github.com/drashland/drash/releases/latest"
            consumer-key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
            consumer-secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
            access-token: ${{ secrets.TWITTER_ACCESS_TOKEN }}
            access-token-secret: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
```

Reply:
Yeah great idea! :D I'll do that for all of the repos too 👍

Reply:
that's a really good idea

Reply:
https://github.com/drashland/dmm/pull/85/commits/b59fc30ff008d68b5db127257475347c793e9341